certified-connectors/AvePoint Cloud Governance/apiProperties.json

```diff
@@ -1,6 +1,36 @@
 {
   "properties": {
     "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "uiDefinition": {
+          "displayName": "Username"
+        },
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "1892da54-7f57-4b6a-b6b0-fdc7f44e04c9",
```

Comment:
@sear-lin-avepoint is it possible to replace this clientId with a dummy data?

Reply:
@Amjed-Ayoub  sorry,  i though it was replaced,  just pushed a commit.